id,module,name,file,doc,markers,status,message,duration
functional_correctness_test_folder/csn/gpt-4.1/no_docstr/test_snippet_24.py::test_case_0,functional_correctness_test_folder.csn.gpt-4.1.no_docstr.test_snippet_24,test_case_0,functional_correctness_test_folder/csn/gpt-4.1/no_docstr/test_snippet_24.py,,xfail,xfailed,,9.59169992711395e-05
functional_correctness_test_folder/csn/gpt-4.1/no_docstr/test_snippet_24.py::test_case_1,functional_correctness_test_folder.csn.gpt-4.1.no_docstr.test_snippet_24,test_case_1,functional_correctness_test_folder/csn/gpt-4.1/no_docstr/test_snippet_24.py,,xfail,xfailed,,8.695799624547362e-05
functional_correctness_test_folder/csn/gpt-4.1/no_docstr/test_snippet_24.py::test_case_2,functional_correctness_test_folder.csn.gpt-4.1.no_docstr.test_snippet_24,test_case_2,functional_correctness_test_folder/csn/gpt-4.1/no_docstr/test_snippet_24.py,,,failed,AttributeError: 'App' object has no attribute 'runtime',8.224998600780964e-05
functional_correctness_test_folder/csn/gpt-4.1/no_docstr/test_snippet_24.py::test_case_3,functional_correctness_test_folder.csn.gpt-4.1.no_docstr.test_snippet_24,test_case_3,functional_correctness_test_folder/csn/gpt-4.1/no_docstr/test_snippet_24.py,,,failed,AttributeError: 'App' object has no attribute 'runtime',7.600002572871745e-05
functional_correctness_test_folder/csn/gpt-4.1/no_docstr/test_snippet_24.py::test_case_4,functional_correctness_test_folder.csn.gpt-4.1.no_docstr.test_snippet_24,test_case_4,functional_correctness_test_folder/csn/gpt-4.1/no_docstr/test_snippet_24.py,,,failed,AttributeError: 'App' object has no attribute 'runtime',6.941601168364286e-05
functional_correctness_test_folder/csn/gpt-4.1/no_docstr/test_snippet_24.py::test_case_5,functional_correctness_test_folder.csn.gpt-4.1.no_docstr.test_snippet_24,test_case_5,functional_correctness_test_folder/csn/gpt-4.1/no_docstr/test_snippet_24.py,,xfail,xfailed,,6.404201849363744e-05
